certification/internal/policy/operator/default.go

```diff
@@ -16,6 +16,9 @@ const (
 	// IndexImageKey is the key in viper that contains the index (catalog) image URI
 	indexImageKey = "indexImage"
 
+	// channel is the key in viper that contains the subscription channel under test
+	channelKeyInViper = "channel"
```

Comment:
NIT: I don't like this variable name, simply because it references the technology we use to manage user-provided configuration by name.

I know this name was chosen in an effort to disambiguate this value with the value of `channelKey` which refers to the key for the channel as found in an operator's annotations. I think there's an opportunity to make this more clear, either by changing this variable (and other config-related variables, for consistency) to be identified in some other way, or changing the annotation variables, or both.

Just my 2c. Up to you on how to proceed.

Reply:
Fair enough. I'll try to come up with a better var name.